libmpv/libmpv.json

```diff
@@ -0,0 +1,169 @@
+{
+    "name": "libmpv",
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/man"
+    ],
+    "buildsystem": "simple",
+    "build-commands": [
+        "python3 waf configure --prefix=/app --enable-libmpv-shared --disable-cplayer --disable-build-date --disable-alsa",
+        "python3 waf build",
+        "python3 waf install"
```

Comment:
1. you shouldn't use `/app`  but `${FLATPAK_DEST}. 
3. you should also use `-j ${FLATPAK_BUILDER_N_JOBS}` so that the build is done in parallel.